### ISSUE TITLE
Upload Lambda Packages

### ISSUE THREAD

**Issue by ktopolov** [mikebowyer/ECE528_Project#6]
Right now, I have been manually copying and pasting code into Lambda. Also, Lambda does not have access to any non-default Python packages (numpy, PIL, etc.). There is a way to upload a ZIP file of code somehow; I will look into this.

If this works, I will be able to save a secondary image with bounding box information.

**Comment by ktopolov** [comment]
This actually seems miserable and we may want to rethink this if possible.

If the website's front end is using Python we will have a much easier time drawing the bounding boxes there I feel. To use `pillow` package in Lambda, which is reqwuired for sdrawing bounding boxes and saving the image, we would need to:  

- Create directory
- Install all packages we want (`pip install pillow`, ...)
- BUT we must be inside of a machine running Amazon Linux. So we would need to do this with an EC2 instance or use docker (which I do not know how to use at all)
- Then we have to ZIP the folder and then upload to S3, then turn it into an AWS Lambda layer and attach it to the Lambda function that needs it 

Basically, the hard part is dealing with the EC2 instance and/or docker.
